### ISSUE TITLE
GoAddTags should not add twice

### ISSUE THREAD

**Issue by gnhuy91** [fatih/vim-go#1064]
Sorry for the bad issue title, couldn't think of a better one.

```
    Number       string `json:"number"`
    TextAllowed  bool
```
### Actual behavior

```
    Number       string `json:"number"` `json:"number"`
    TextAllowed  bool `json:"text_allowed"`
```
### Expected behavior

```
    Number       string `json:"number"`
    TextAllowed  bool `json:"text_allowed"`

```

**Comment by fatih** [comment]
Hi @gnhuy91 

Thanks for the bug report. This is one of the cases where an AST based tool is needed or things get complex easily. Let's tackle this as well with the other `:GoAddTags` related issues.

**Comment by fatih** [comment]
Fixed with #1204 please, feel free to test and reopen if it's not the case.
